src/app/config/header.js

```diff
@@ -0,0 +1,15 @@
+
+const header = {
```

Comment:
@kudzayimutunga unnecessary white spacing, could also just export instead of defining and exporting separately.